### PR TITLE
Provides several UX improvements for order pages and allows to change order addresses

### LINKED ISSUE #16039: Improvements for Order view
Improvements needed for Order view page:

1. Material icon ‘account_box’ is missing
2. Edit should be done through material icon ‘more_vert ‘ —>  Edit  
3. Price background is missing
4. Add icons in tabs
5. Increase spacing between buttons to 15px 
6. Text color in buttons should be #576C72

order_feedback

Regarding order preivew : 

- The 'open order' button is a little bit too close to the line below, which can lead to misclick.

order_preview_feedback

- The order line in order list should be clickable as it was in legacy page. 
- The status hover in order list should have the following behavior: 

Capture d’écran 2019-10-21 à 14 10 53

Edit Adress

Mockups for the poppin: https://invis.io/YKTGGFZAMCB#/382686482__Order_Details_-_Change_Adress

@TristanLDD I'm opening issue with your requests, so it won't get lost. :wink:

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Provides several UX improvements for order pages and allows to change order addresses
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/16039
| How to test?  | Please see ticket. Please rebuild assets.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16570)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/pages/order/view.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ $(() => {
   handlePrivateNoteChange();
   handleUpdateOrderStatusButton();
 
+
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/pages/order/view.js]: Remove empty line

File: admin-dev/themes/new-theme/js/pages/order/view.js [hunk 2]
```diff
@@ -49,6 +50,8 @@ $(() => {
 
   initAddCartRuleFormHandler();
   initAddProductFormHandler();
+  initChangeAddressFormHandler();
+
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/pages/order/view.js]: Remove empty line

**Comment by matks** [comment]
@Progi1984 Empty lines removed ✅and PR rebased

**Review by Progi1984 — DISMISSED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @matks !

There is still some points listed in the issue that are KO : 

- Text color in buttons should be #576C72
- The status hover in order list should have the following behavior (see issue for the behaviour, here a screenshot of actual behaviour)
pr16570 arrow
- The change address popin should have a bigger dropdown. See mockup : https://invis.io/YKTGGFZAMCB#/382686482__Order_Details_-_Change_Adress
And screenshot : 
pr16570 change address

Bisous !

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig [hunk 3]
```diff
@@ -84,25 +86,36 @@
...
+                href="{{ getAdminLink('AdminAddresses', true, {
+                  'realedit': 1,
+                  'addaddress': 1,
+                  'address_type': 1,
+                  'id_order': orderForViewing.id,
+                  'id_address': orderForViewing.shippingAddress.addressId,
+                  'back': path('admin_orders_view', {'orderId': orderForViewing.id})
+                }) }}"
+              >
+                {{ 'Edit address'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig]:
```suggestion
                {{ 'Edit address'|trans({}, 'Admin.Actions') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig [hunk 4]
```diff
@@ -84,25 +86,36 @@
...
+              </a>
+
+              <a href="#"
+                class="dropdown-item js-update-customer-address-modal-btn"
+                data-toggle="modal"
+                data-target="#updateCustomerAddressModal"
+                data-address-type="shipping"
+              >
+                {{ 'Change address'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig]:
```suggestion
                {{ 'Change address'|trans({}, 'Admin.Actions') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/update_customer_address_modal.html.twig [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+ *#}
+
+<div class="modal fade" id="updateCustomerAddressModal">
+  {{ form_start(changeOrderAddressForm,
+    {'action': path('admin_orders_change_customer_address', {'orderId': orderForViewing.id, 'customerId': orderForViewing.customer.id})}) }}
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">
+            {{ 'Change address'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/update_customer_address_modal.html.twig]:
```suggestion
            {{ 'Change address'|trans({}, 'Admin.Actions') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig [hunk 6]
```diff
@@ -126,25 +139,37 @@
...
+                href="{{ getAdminLink('AdminAddresses', true, {
+                  'realedit': 1,
+                  'addaddress': 1,
+                  'address_type': 2,
+                  'id_order': orderForViewing.id,
+                  'id_address': orderForViewing.invoiceAddress.addressId,
+                  'back': path('admin_orders_view', {'orderId': orderForViewing.id})
+                }) }}"
+              >
+                {{ 'Edit address'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig]:
```suggestion
                {{ 'Edit address'|trans({}, 'Admin.Actions') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig [hunk 7]
```diff
@@ -126,25 +139,37 @@
...
+              </a>
+
+              <a href="#"
+                class="dropdown-item js-update-customer-address-modal-btn"
+                data-toggle="modal"
+                data-target="#updateCustomerAddressModal"
+                data-address-type="invoice"
+              >
+                {{ 'Change address'|trans({}, 'Admin.Orderscustomers.Feature') }}
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig]:
```suggestion
                {{ 'Change address'|trans({}, 'Admin.Actions') }}
```

**Comment by Robin-Fischer-PS** [comment]
Thanks @NeOMakinG !

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks & @NeOMakinG !

**Pull request merged**